### DISCUSSION TITLE
CLI Help

### DISCUSSION THREAD

**Discussion by jsaxx** [glzr-io/glazewm#779] [Q&A]
Hello,

I would like to integrate in a script glazewm commads for example `glazewm.exe query workspaces`. But when running it, I am getting no response. I have tried `glazewm.exe query workspaces > test.txt` but again nothing gets output.

Can I plese get any pointer on how to get the response from this (or any other) command? TIA

**Comment by rsa17826** [comment]
you have to use the exe in the cli folder - "C:\Program Files\glzr.io\GlazeWM\cli\glazewm.exe" not "C:\Program Files\glzr.io\GlazeWM\glazewm.exe"

**Reply by lars-berger** [reply]
> you have to use the exe in the cli folder - "C:\Program Files\glzr.io\GlazeWM\cli\glazewm.exe" not "C:\Program Files\glzr.io\GlazeWM\glazewm.exe"

From v3.8.0 onwards, there's a bug fix that made it so that `glazewm query workspaces` works now without having to explicitly run the CLI executable. Under the hood, we're adding the CLI executable to the `PATH` env variable, and if called with args/flags that should be forwarded to the main executable, it'll then launch that.

![image](https://github.com/user-attachments/assets/b77034e6-b9e9-418d-b418-4fb4b09bac85)

So the original issue @jsaxx mentioned should hopefully not be present on newer versions 👍
